catalog.IPTC-G2-Standards_37.xml

```diff
@@ -0,0 +1,657 @@
+        <definition xml:lang="en-GB">The category code of the story may be any of the single alpha characters from the vocabulary. (ANPA 1312 specs, section J)</definition>
+        <name xml:lang="en-GB">ANPA 1312 Category Codes</name>
+        <note xml:lang="en-GB">This vocabulary is owned by NAA - www.naa.org</note>
+    </scheme>
+    <scheme alias="a1312prio" uri="http://cvx.iptc.org/anpa1312-priority/" modified="2011-12-15T12:00:00+00:00">
+        <definition xml:lang="en-GB">The priority of a transmission is designated by any of the single alpha characters from the vocabulary.  (ANPA 1312 specs, section H)</definition>
+        <name xml:lang="en-GB">ANPA 1312 Priority Codes</name>
+        <note xml:lang="en-GB">The vocabulary is owned by NAA - www.naa.org</note>
+    </scheme>
+    <scheme alias="a1312svc" uri="http://cvx.iptc.org/anpa1312-servicelevel/" modified="2011-12-15T12:00:00+00:00">
+        <definition xml:lang="en-GB">The wire designator code will be an upper- or lowercase single alpha character that identifies the basic nature of the service. (ANPA 1312 specs, section B)</definition>
+        <name xml:lang="en-GB">ANPA 1312 Service Level Codes</name>
+        <note xml:lang="en-GB">The vocabulary is owned by NAA - www.naa.org</note>
+    </scheme>
+    <scheme alias="a1312vers" uri="http://cvx.iptc.org/anpa1312-version/" modified="2011-12-15T12:00:00+00:00">
+        <definition xml:lang="en-GB">The information in this field is designed to enable newspaper systems to distinguish new leads, inserts, corrections and adds, so that the process of automatically assembling separately transmitted portions of a story is not limited to putting the latest material at the bottom of the file. (ANPA 1312 specs, section O)</definition>
+        <name xml:lang="en-GB">ANPA 1312 Version Codes</name>
+        <note xml:lang="en-GB">The vocabulary is owned by NAA - www.naa.org</note>
+    </scheme>
+    <scheme alias="bcp47" uri="http://cvx.iptc.org/bcp47/" authority="https://www.ietf.org/" modified="2021-09-17T12:00:00+00:00">
+        <definition xml:lang="en-GB">Language tags are used to help identify languages, whether spoken, written, signed, or otherwise signaled, for the purpose of communication. </definition>
+        <name xml:lang="en-GB">BCP47 Language Tags</name>
+        <note xml:lang="en-GB">Body owning this CV: IETF - www.ietf.org</note>
+    </scheme>
+    <scheme alias="cusip" uri="http://cvx.iptc.org/cusip/" authority="https://www.cusip.com/" modified="2021-09-17T12:00:00+00:00">
+        <definition xml:lang="en-GB">The CUSIP is the National Securities Identification Number for products issued from both the United States and Canada.</definition>
+        <name xml:lang="en-GB">Committee on Uniform Security Identification Procedures</name>
+        <note xml:lang="en-GB">The codes are owned by the American Bankers Association, and operated by S&amp;P Capital IQ.</note>
```

Comment:
Two separate notes:
Body owning this CV: Owned by the American Bankers Association, and operated by S&P Capital IQ.
Body owning the format of the codes: CUSIP Global Services (CGS) - http://www.cusip.com/